app/integrations/discord/interactions/resolvers/ping.rb

```diff
@@ -0,0 +1,28 @@
+module Discord
+  module Interactions
+    module Resolvers
+      class Ping < Resolver
+        EXPECTED_GLOBAL_NAME = 'Discord'.freeze
+        PONG_RESPONSE_TYPE = 1
```

Comment:
why is Pong response type assigned to 1?  What other response types will we have? Is this something Discord is stating or is it us?

Reply:
we are also moving `EXPECTED_GLOBAL_NAME` to `Interactions::Requestcomponents::User` *resources

Reply:
Done!